extensions/amp-ad-network-adsense-impl/0.1/adsense-a4a-config.js

```diff
@@ -41,28 +48,71 @@ export const URL_EXPERIMENT_MAPPING = {
   '0': null,
 };
 
+export const ADSENSE_EXPERIMENTS = {
+  UNCONDITIONED_CANONICAL_EXP: '21062154',
+  UNCONDITIONED_CANONICAL_CTL: '21062155',
+  CANONICAL_EXP: '21062158',
+  CANONICAL_CTL: '21062159',
+};
+
+export const ADSENSE_EXP_NAMES = {
+  UNCONDITIONED_CANONICAL: 'expAdsenseUnconditionedCanonical',
+  CANONICAL: 'expAdsenseCanonical',
+};
+
 /**
+ * Attempts to select into Adsense experiments.
  * @param {!Window} win
  * @param {!Element} element
- * @param {boolean} useRemoteHtml
- * @return {boolean}
  */
-export function adsenseIsA4AEnabled(win, element, useRemoteHtml) {
-  if (useRemoteHtml || !isGoogleAdsA4AValidEnvironment(win) ||
-      !element.getAttribute('data-ad-client')) {
-    return false;
-  }
+function selectExperiments(win, element) {
+  selectAndSetExperiments(win, element,
+      [ADSENSE_EXPERIMENTS.UNCONDITIONED_CANONICAL_EXP,
+        ADSENSE_EXPERIMENTS.UNCONDITIONED_CANONICAL_CTL],
+      ADSENSE_EXP_NAMES.UNCONDITIONED_CANONICAL,
+      true);
+
+
   // See if in holdback control/experiment.
   let experimentId;
   const urlExperimentId = extractUrlExperimentId(win, element);
   if (urlExperimentId != undefined) {
```

Comment:
Can you remind me why we do this undefined check?  I assume its so we can support "falsey" values like empty string or 0 (though this is a string so '0' would evaluate to true).  Can we change this code to:

const experimentId = URL_EXPERIMENT_MAPPING[extractUrlExperimentId(win, element)];
if (experimentId) {
   addExperiment...
}

Reply:
done